lib/commons/aria/get-owned-virtual.js

```diff
@@ -0,0 +1,13 @@
+/* global aria, dom */
+aria.getOwnedVirtual = function getOwned({ actualNode, children }) {
+	if (!actualNode || !children) {
+		throw new Error('getOwnedVirtual requires a virtual node');
+	}
+	return dom.idrefs(actualNode, 'aria-owns').reduce((ownedElm, element) => {
```

Comment:
Renamed. This should have been called `ownedElms`, since this is an array. I do prefer giving the name of the thing I'm building up in the reduce function.